src/components/view-center.js

```diff
@@ -0,0 +1,48 @@
+import { x } from '@xstyled/styled-components'
+import BancoComida from '../iamges/banco-de-comida.jpeg'
+import { useNavigate } from 'react-router-dom';
+
+export default function ViewCenter() {
+    const navigate = useNavigate();
+    return (
+            <x.div 
```

Comment:
I think that with these props it's enough

```suggestion
            <x.div 
              w="100%"
              p={{ _: '17px', lg: '23px' }}
              display="flex"
              boxShadow="0px 4px 20px rgba(0, 0, 0, 0.1)"
              borderRadius="20px"
              gap="26px"
            >
```